CMakeLists.txt

```diff
@@ -287,23 +300,58 @@ set(HEADER_FILES
 link_directories( $ENV{CUDA_LIB} )
 include_directories( $ENV{CUDA_INC} )
 
+# find spdlog lib
+if(NOT TARGET spdlog)
+    # Stand-alone build
+    find_package(spdlog)
+endif()
+if(NOT TARGET spdlog) # use local fallback
```

Comment:
same as line 304?

Reply:
In line 304 CMake checks if the user already included spdlog (maybe to include in a specific location) bevor and if not uses a globally installed on.
In line 308 CMake checks if the previous include of spdlog failed and if so uses the local shipped version.